pkg/cmd/ssh_gcp.go

```diff
@@ -71,7 +77,13 @@ func sshToGCPNode(nodeName, path, user, pathSSKeypair string, sshPublicKey []byt
 	time.Sleep(45 * time.Second)
 
 	key := filepath.Join(pathSSKeypair, "key")
-	sshCmd := fmt.Sprintf("ssh -i " + key + " -o \"ProxyCommand ssh -W %%h:%%p -i " + key + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no " + bastionNode + "\" " + node + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no")
+	var sshCmd string
+	if debugSwitch {
+		sshCmd = fmt.Sprintf("ssh -v -i " + key + " -o \"ProxyCommand ssh -W %%h:%%p -i " + key + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no " + bastionNode + "\" " + node + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no")
+	} else {
+		sshCmd = fmt.Sprintf("ssh -i " + key + " -o \"ProxyCommand ssh -W %%h:%%p -i " + key + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no " + bastionNode + "\" " + node + " -o IdentitiesOnly=yes -o StrictHostKeyChecking=no")
+	}
+
 	fmt.Println(sshCmd)
 	cmd := exec.Command("bash", "-c", sshCmd)
```

Comment:
please, if you already touch this code, directly call ssh instead of bash

Reply:
I didn't  touch this code here, only change just add -v for ssh verbose

Reply:
Ok I see. I will remove it seem just find bash in here.

Reply:
fixed, and we have a dedicate issue for remove bash  https://github.com/gardener/gardenctl/issues/267 but it seems not start yet.

Reply:
I know, but if you touch this code anyhow it makes sense to directly fix at least this part